### DISCUSSION TITLE
App design

### DISCUSSION THREAD

**Discussion by YuriDevAT** [YurisCodingClub/dev-cv#17] [Ideas]
# Design of the app

How should the design of the app look like?
Let's collect some ideas (maybe from [Behance](https://www.behance.net/) or [dribbble](https://dribbble.com/)) and choose with which kind of design we want to go further.

- [ ] Design for the form components
- [ ] Design for page layout (components and CV preview)

**Comment by MusfiqDehan** [comment]
Hey @YuriDevAT,

I am adding a basic design idea for the `DEV-CV` . Please let me know, what you think about this simple design! Should it have a landing page first? Then after clicking a button on the landing page, users will go to the original page.

![design-idea-dev-CV](https://user-images.githubusercontent.com/47440165/194779944-3836a78b-03d1-4c7a-a466-f7bd1b84965d.png)

**Comment by YuriDevAT** [comment]
I like the layout structure. I think for now, now landing page is needed and the user should directly land on the form page and can start working on their CV. Well done @MusfiqDehan! 🚀

**Comment by DugarRishab** [comment]
Hi, I would like to give a suggestion here. I really liked the form interface of [app.flowcv.com](app.flowcv.com). Something like this ( but better ) would be awesome.

**Comment by YuriDevAT** [comment]
We should also take into consideration how the layout structure should look like on a mobile version. For now, @MusfiqDehan suggestion is pretty neat for desktop, as a User, you can see the form you are adding your content and at the same time, how the CV will look like.

**Comment by utkarsh1311** [comment]
Any progress on the design part?

**Reply by MusfiqDehan** [reply]
Please read the instructions on the organization's homepage. https://github.com/YurisCodingClub
But if you have any design ideas, feel free to share them here.
